Anchor.toml

```diff
@@ -19,15 +20,18 @@ marginfi = "MFv2hWf31Z9kbCa1snEPYctwafyhdvnV7FZnsebVacA"
 url = "https://api.apr.dev"
 
 [provider]
-cluster = "localnet"
-# cluster = "https://devnet.rpcpool.com/"
+cluster = "Localnet"
 wallet = "~/.config/solana/id.json"
 
+# (remove RUST_LOG= to see bankRun logs)
 [scripts]
-test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/*.spec.ts --exit --require tests/rootHooks.ts"
+test = "RUST_LOG= yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/*.spec.ts --exit --require tests/rootHooks.ts"
+
+# Staked collateral tests only
+# test = "RUST_LOG= yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/s*.spec.ts --exit --require tests/rootHooks.ts"
```

Comment:
Any alternative location where they can stay? As the TS suite grows it can take several minutes to run and it's nice to have a documented method to run a portion of the suite. Most programs do not support running a portion of anchor tests, so generally people would be unfamiliar with that option unless it's documented.